src/transformers/generation_utils.py

```diff
@@ -725,6 +729,10 @@ def _get_logits_processor(
             if exponential_decay_length_penalty is not None
             else self.config.exponential_decay_length_penalty
         )
+        suppress_tokens = suppress_tokens if suppress_tokens is not None else self.config.suppress_tokens
```

Comment:
Both `suppress_tokens` and `begin_suppress_tokens` need to be added to all the official model configs: https://huggingface.co/openai/whisper-medium.en/blob/main/config.json (as discussed on Slack)

Reply:
Yes! Should I do that before merging?

Reply:
Definitely :-)